pyk/src/pyk/kcfg.py

```diff
@@ -1,5 +1,6 @@
 import json
 from abc import ABC, abstractmethod
+from collections import OrderedDict
```

Comment:
`dict` is insertion ordered as of Python 3.7, so this is probably not necessary.